mlir/include/mlir/Interfaces/CallInterfaces.td

```diff
@@ -31,7 +73,7 @@ def CallOpInterface : OpInterface<"CallOpInterface"> {
   }];
   let cppNamespace = "::mlir";
 
-  let methods = [
+  let methods = !listconcat(ArgumentAttributesMethods.methods, [
```

Comment:
You should be able to just use the operator for this `] # ArgumentAttributesMethods.methods`